pkg/services/accesscontrol/accesscontrolmock/mock.go

```diff
@@ -0,0 +1,153 @@
+package accesscontrolmock
```

Comment:
```suggestion
package mock
```

The path will tell you that it belongs to access control.

Reply:
I've recently been convinced by @joanlopez to switch to this naming convention :smile:
The point in having this clearly specific name was to avoid conflicts with other mock sub-packages which would result in aliasing.
That being said, I'm not reluctant to the other naming convention, where the path gives you the idea.

Reply:
I see where that's coming from, but I think having naming conflicts for internal packages is a feature. It discourages importing non-domain package packages from other services.